doc/source/rllib/rllib-offline.rst

```diff
@@ -250,20 +695,7 @@ To load sample data using Dataset, specify input and input_config keys like the
 	...
     }
 
-To write sample data to JSON or Parquet files using Dataset, specify output and output_config keys like the following:
-
-.. code-block:: python
 
-    config = {
-        "output": "dataset",
-        "output_config": {
-            "format": "json",  # json or parquet
-	    # Directory to write data files.
-            "path": "/tmp/test_samples/",
-	    # Break samples into multiple files, each containing about this many records.
-            "max_num_samples_per_file": 100000,
-        }
-    }
 
 Writing Environment Data
```

Comment:
Dumb question: Are these paragraphs still needed (new API stack)?
If not (or if they only refer to old API stack stuff), let's completely remove them.

Reply:
Same question for the above `Ray Data Integration` section.

Reply:
Yes they are all old API stack and will be removed by my soon. They helped me structuring the file better.